x-pack/test/api_integration/apis/cloud_security_posture/status/status_unprivileged.ts

```diff
@@ -194,14 +282,53 @@ export default function (providerContext: FtrProviderContext) {
           .auth(UNPRIVILEGED_USERNAME, 'changeme')
           .expect(200);
 
-        expect(res.kspm.status).to.be('unprivileged');
-        expect(res.cspm.status).to.be('unprivileged');
-        expect(res.vuln_mgmt.status).to.be('not-installed');
+        expect(res.kspm.status).to.eql(
+          'unprivileged',
+          `expected unprivileged but got ${res.kspm.status} instead`
+        );
+        expect(res.cspm.status).to.eql(
+          'unprivileged',
+          `expected unprivileged but got ${res.cspm.status} instead`
+        );
+        expect(res.vuln_mgmt.status).to.eql(
+          'not-installed',
+          `expected not-installed but got ${res.vuln_mgmt.status} instead`
+        );
 
-        expect(res.indicesDetails[0].status).to.be('unprivileged');
-        expect(res.indicesDetails[1].status).to.be('empty');
-        expect(res.indicesDetails[2].status).to.be('unprivileged');
-        expect(res.indicesDetails[3].status).to.be('empty');
+        expect(
+          res.indicesDetails.find((idx) => idx.index === LATEST_FINDINGS_INDEX_DEFAULT_NS)?.status
+        ).to.eql(
+          'unprivileged',
+          `expected unprivileged but got ${
+            res.indicesDetails.find((idx) => idx.index === LATEST_FINDINGS_INDEX_DEFAULT_NS)?.status
+          } instead`
+        );
+        expect(
+          res.indicesDetails.find((idx) => idx.index === FINDINGS_INDEX_PATTERN)?.status
+        ).to.eql(
+          'empty',
+          `expected empty but got ${
+            res.indicesDetails.find((idx) => idx.index === FINDINGS_INDEX_PATTERN)?.status
+          } instead`
+        );
+        expect(
+          res.indicesDetails.find((idx) => idx.index === BENCHMARK_SCORE_INDEX_DEFAULT_NS)?.status
+        ).to.eql(
+          'unprivileged',
+          `expected unprivileged but got ${
+            res.indicesDetails.find((idx) => idx.index === BENCHMARK_SCORE_INDEX_DEFAULT_NS)?.status
+          } instead`
+        );
+        expect(
+          res.indicesDetails.find((idx) => idx.index === LATEST_VULNERABILITIES_INDEX_DEFAULT_NS)
+            ?.status
+        ).to.eql(
+          'empty',
+          `expected empty but got ${
+            res.indicesDetails.find((idx) => idx.index === LATEST_VULNERABILITIES_INDEX_DEFAULT_NS)
+              ?.status
+          } instead`
+        );
```

Comment:
I suggest adding the checked index to the message
```suggestion
        expect(
          res.indicesDetails.find((idx) => idx.index === LATEST_VULNERABILITIES_INDEX_DEFAULT_NS)
            ?.status
        ).to.eql(
          'empty',
          `expected ${LATEST_VULNERABILITIES_INDEX_DEFAULT_NS} empty but got ${
            res.indicesDetails.find((idx) => idx.index === LATEST_VULNERABILITIES_INDEX_DEFAULT_NS)
              ?.status
          } instead`
        );
```

Reply:
And to clean up the code you can create the following assertion func:

```ts
const assertIndexStatus(indicesDetails, indexName, status) => {
  const actualStatus = indicesDetails.find(idx => idx.name === indexName)?.status === status);
  expect(actualStatus).to.eql(status, `expected ${indexName} to be ${status} but got  ${actualStatus} instead`);
}
```

Reply:
And use it
```ts
assertIndexStatus(res.indicesDetails, LATEST_VULNERABILITIES_INDEX_DEFAULT_NS, 'empty');
```